README.md

```diff
@@ -146,13 +146,13 @@ const { flags, values, positionals } = parseArgs(argv, options);
 ### F.A.Qs
 
 - Is `cmd --foo=bar baz` the same as `cmd baz --foo=bar`?
-  - Yes, if `withValue: ['foo']`, otherwise no
+  - Yes
```

Comment:
This is the only answer with the drop cap

```suggestion
  - yes
```

Reply:
Fixed, thanks